source/users/enable-user-metadata.txt

```diff
@@ -226,13 +226,10 @@ Procedure
 Access User Metadata from a Client Application
 ----------------------------------------------
 
-.. tabs-realm-sdks::
-
-   .. tab::
-      :tabid: ios
-
-      .. tip:: Read a User's Metadata
-         
-         Learn how to :ref:`read a user's metadata from an iOS
-         application <ios-read-user-metadata>`.
+Realm SDKs have different implementations for accessing user metadata in a client
+application. Check out the details in the SDK documentation:
 
+- :ref:`Realm Swift SDK <ios-read-user-metadata>`
+- :ref:`Realm Flutter SDK <flutter-read-user-metadata>`
+- :ref:`Realm .NET SDK <dotnet-read-user-metadata>`
+- :ref:`Realm Node.js SDK <ios-read-user-metadata>`
```

Comment:
is this Realm Node.js SDK link purposefully pointing to a duplicate `<ios-read-user-metadata>`?

Reply:
lol. No. I thought I fixed that, but clearly didn't. Thanks for pointing it out!